src/main/java/nextstep/sessions/domain/ApplicationStatusEnum.java

```diff
@@ -0,0 +1,22 @@
+package nextstep.sessions.domain;
+
+import java.util.Arrays;
+
+public enum ApplicationStatusEnum {
+    apply("00"),
+    approve("01"),
+    cancel("99");
```

Comment:
`Enum`의 경우 상수이므로 대문자 표기가 올바른 컨벤션일 것 같습니다 😄 
그리고 소소한 의견이긴 한데 `Enum`을 끝에 붙이지 않은 
`ApplicationStatus`도 나쁘지 않을 것 같습니다 🙇 
```suggestion
    APPLY("00"),
    APPROVE("01"),
    CANCEL("99");
```